troposphere/static/js/components/projects/list/Project.react.js

```diff
@@ -27,17 +27,24 @@ define(function (require) {
     renderForRouter: function () {
       var project = this.props.project;
       return (
-        <li className={"project-card " + this.props.className}>
-          <Router.Link to="project-resources" params={{projectId: project.id}}>
-            {this.renderBody()}
-          </Router.Link>
+        <li className={"col-md-4" + this.props.className} style={{padding: "15px"}}>
```

Comment:
So we don't mean there to be a space between `col-md-4` and `this.props.classname` - right? 
We could use a template string here:
```
<li className={`col-md-4${this.props.className}`} ... 
```

Or do we want to use [classnames](https://github.com/JedWatson/classnames#usage) to define this?